h2o-algos/src/jmh/java/hex/pca/PCAJMH.java

```diff
@@ -0,0 +1,92 @@
+package hex.pca;
+
+import hex.DataInfo;
+import water.DKV;
+import water.Key;
+import water.fvec.Frame;
+import water.util.FrameUtils;
+
+import static hex.pca.JMHConfiguration.logLevel;
+import static hex.pca.PCAModel.PCAParameters.Method.GramSVD;
+import static water.TestUtil.parse_test_file;
+import static water.TestUtil.stall_till_cloudsize;
+
+/**
+ * @author mathemage <ha@h2o.ai>
+ *         created on 24.6.17
+ */
+public class PCAJMH {
+	
+	PCAModel.PCAParameters paramsQuasar;
```

Comment:
Check formatting here please, it looks like too many spaces for indentation.

Reply:
@mmalohlava OK, I have changed the file to 2-space indentation.